kedro-datasets/kedro_datasets/geopandas/geojson_dataset.py

```diff
@@ -50,12 +58,13 @@ class GeoJSONDataset(
     """
 
     DEFAULT_LOAD_ARGS: dict[str, Any] = {}
-    DEFAULT_SAVE_ARGS = {"driver": "GeoJSON"}
+    DEFAULT_SAVE_ARGS: dict[str, Any] = {}
```

Comment:
I don't know why this was required, because it also works with shapefiles out of the box. It looks like `fiona` finds the driver automatically by the file name.

Reply:
> It looks like fiona finds the driver automatically by the file name.

Most likely!